packages/jest-snapshot/dts/natural-compare.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'natural-compare' {
```

Comment:
Send PR to definitelytyped?

Reply:
Sure, I was thinking about it but I forgot it 🙈